resources/schema.py

```diff
@@ -510,6 +510,49 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return CreatePierMutation(pier=pier)
 
 
+class UpdatePierMutation(graphene.ClientIDMutation):
+    class Input(PierInput):
+        id = graphene.ID(required=True)
+
+    pier = graphene.Field(PierNode)
+
+    @classmethod
+    @login_required
+    @superuser_required
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **kwargs):
+        # TODO: Should check if the user has permissions to
+        # delete the specific resource
+        id = from_global_id(kwargs.pop("id"))[1]
+
+        try:
+            harbor_id = kwargs.pop("harbor_id", None)
+            if harbor_id:
+                global_harbor_id = from_global_id(harbor_id)[1]
```

Comment:
well, this is actually `harbor_id`, but the var above could be called `global_harbor_id` 🙂